src/script/lua_api/l_noise.h

```diff
@@ -150,7 +151,8 @@ class LuaPcgRandom : public ModApiBase
 
 public:
 	LuaPcgRandom(u64 seed) : m_rnd(seed) {}
-	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed, seq) {}
+	// @TODO: set seq
+	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed) {}
```

Comment:
That's not what I meant:
If we switch from pcg to mt for random numbers, the lua api will return different numbers for the same seed value, thus potentially breaking mods that rely on this.